pom.xml

```diff
@@ -125,4 +125,15 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+    <repositories>
+        <repository>
```

Comment:
Do we need this here, isn't it something that can be activated in the `.m2/settings.xml` ?